.jenkins/jenkins_test1.sh

```diff
@@ -9,6 +9,6 @@ source ~/.bashrc
 
 # Test theano core
 PARTS="theano/compat theano/compile theano/d3viz theano/gof theano/misc theano/sandbox/linalg theano/sandbox/tests theano/scalar theano/scan_module theano/sparse theano/tensor theano/tests theano/typed_list"
-THEANO_PARAM="theano --with-timer --timer-top-n 10"
+THEANO_PARAM="${PARTS} --with-timer --timer-top-n 10"
```

Comment:
The gpuarray tests were completing in under 20 mins, while the theano cpu + old backend took almost 2 hours.

The goal is to move the old backend to the same job as the new backend to split time more equally. I forgot to correct that comment. Of course, there are some gpu tests outside the backend folders, hence why init_gpu_device is still set.